src/util/system.h

```diff
@@ -336,6 +326,16 @@ class ArgsManager
      */
     std::string GetArg(const std::string& strArg, const std::string& strDefault) const;
 
+    /**
```

Comment:
Do we really need to change the order of methods in the header file? This makes the review unnecessarily harder than it has to be.